services/web/src/utils/date.js

```diff
@@ -24,3 +24,48 @@ function getFormatted(arg, locale, options) {
   const date = typeof arg === 'string' ? new Date(arg) : arg;
   return new Intl.DateTimeFormat(locale, options).format(date);
 }
+
+export function fromNow(
+  date,
+  nowDate = Date.now(),
+  rft = new Intl.RelativeTimeFormat(undefined, { numeric: 'auto' })
```

Comment:
`rtf`?

also is there a reason not to use [luxon](https://moment.github.io/luxon/api-docs/index.html#datetimetorelative) here?

Reply:
rtf = RelativeTimeFormat 

this allow us to localize the output e.g. you can  pass `new Intl.RelativeTimeFormat('de', { numeric: 'auto' })`, it will say `X Tagen` instead of `X Days`

Re luxon we are not include it currently, and if this all we need, i dont see reason to include it.

Reply:
> rtf = RelativeTimeFormat
ah yeah I know I meant it's `rft` there

for luxon I dunno ... it seems like we now have a need which is something it does out of the box, uses all the latest APIs under the hood, is tested, and has fallbacks... most current projects are using luxon now, we almost always end up needing it.. I'd say it's time to just bring it into the fold

Reply:
ok fair enough